libkbfs/block_retrieval_queue.go

```diff
@@ -371,10 +371,13 @@ func (brq *blockRetrievalQueue) CacheAndPrefetch(ctx context.Context,
 	return nil
 }
 
+// checkCaches returns a block if it's in one of our caches. If it's
+// successful, it triggers CacheAndPrefetch. Otherwise, it doesn't trigger it
```

Comment:
Is this comment out of date?  I don't see where `checkCaches` calls `CacheAndPrefetch` at all now.

Reply:
Yep, I'll update it.